### PR TITLE
Fixed some typos and added a few wording edits

### PR DESCRIPTION
**Pull request by maxdrohde**

### CODE REVIEW

File: vignettes/gghighlight.Rmd [hunk 1]
```diff
@@ -160,13 +160,13 @@ Note that, in this case, `label_key = type` is needed to show labels because `gg
...
 
-To construct a predicate expression like bellow, we need to determine a threshold (in this example, `20`). But it is difficult to choose a nice one before we draw plots.
+To construct a predicate expression like below, we need to determine a threshold (in this example, `20`). But it is difficult to choose a nice one before we draw plots.
 
 ```{r predicate-example, eval=FALSE}
 max(value) > 20
 ```
 
-So, `gghighlight()` allows predicates that return non-logical (e.g. numeric and character) results. The values are used for sorting data and the top `max_highlight` of rows/groups are highlighted:
+So, `gghighlight()` allows predicates that return non-logical (e.g., numeric and character) results. The values are used for sorting data and the top `max_highlight` of rows/groups are highlighted:
```
Comment (yutannihilation) [inline vignettes/gghighlight.Rmd:169]: > In British English, “i.e.” and “e.g.” are *not* followed by a comma

https://jakubmarian.com/comma-after-i-e-and-e-g/

**Review by yutannihilation — CHANGES_REQUESTED** [review]
Thanks! Honestly, I'm very poor at English, so this kind of contribution is really appreciated :)

However, a few of them seem correct in British English, which ggplot2 uses. Could you revert them?

**Comment by yutannihilation** [comment]
Thanks so much!

**Comment by maxdrohde** [comment]
Glad to help! I've enjoyed using your `gghighlight` package.

Thanks for pointing out those differences between American and British English -- I'll keep that in mind for the future.

**Pull request merged**